src/tic_tac_toe/GameBoard.java

```diff
@@ -62,12 +63,18 @@ public Boolean checkIfValidMove(String user, String[] board) {
         return board[Integer.parseInt(user) - 1].equals("_ ");
     }
 
-    public void updateBoardUser(String move) {
-        board[Integer.parseInt(move) - 1] = "X ";
-    }
+    public void updateBoard(String move, String whoseMove) {
+
+        String whichPlayer = whoseMove.toLowerCase();
 
-    public void updateBoardComputer(String move) {
-        board[Integer.parseInt(move) - 1] = "O ";
+        if (!Objects.equals(whichPlayer, "computer") &&
```

Comment:
- Took updateBoardUser() and updateBoardComputer() and refactored them into one updateBoard() method since they do essentially the same logic except add an X or O to the board. Trying to stick to Do not repeat yourself (DRY) principles